### ISSUE TITLE
Bug in example listbox/listbox-scrollable.html: "aria-selected" never set on options

### ISSUE THREAD

**Issue by spectranaut** [w3c/aria-practices#813]
The documentation of the [listbox-scrollable](https://w3c.github.io/aria-practices/examples/listbox/listbox-scrollable.html) example claims the `aria-selected` attribute should be set on the option that is visually selected, but it is never set.

**Comment by spectranaut** [comment]
This bug was actually already reported and fixed by https://github.com/w3c/aria-practices/pull/795 -- thanks @tryenc !
